modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/utils/dataUtils.ts

```diff
@@ -5,6 +5,86 @@
 
 type LocalUIData = APISchemaUIData | APIApplicationUIData;
 
+interface FieldsDataToProperties {
```

Comment:
@cgmonte Please, rename it to `AddObjectFieldsDataToPropertiesProps` to keep the pattern.